APITest/APITest/Controllers/EmployeeController.cs

```diff
@@ -1,23 +1,77 @@
-﻿using RestSharp;
-using System.Threading.Tasks;
+﻿using System.Threading.Tasks;
+using RestSharp;
+using APITest.Constants;
 
 namespace APITest.Controllers
 {
-    public class EmployeeController : BaseController
+    public class EmployeeController
     {
-        private const string GetEmployeeUrl = "/employees";
-        private const string GetEmployeeByIdUrl = "/employees/{0}";
+        private async Task AddCookiesInRequest(RestRequest request)
+        {
+            var response = await GetAllEmployeesAsync();
+
+            request.AddCookie("PHPSESSID", response.Cookies[0].Value);
+        }
+
+        public async Task<IRestResponse> GetAllEmployeesAsync()
+        {
+            var client = new RestClient(ConfigConstants.GetAllEmployeesURL);
+
+            var request = new RestRequest(ConfigConstants.GetAllEmployeesURL, Method.GET);
+
+            return await Task.Run(() => client.Get(request));
+        }
+
+        public async Task<IRestResponse> GetEmployeeByIdAsync(uint employeeId)
+        {
+            var client = new RestClient(string.Concat(ConfigConstants.GetEmployeeURL, employeeId));
+
+            var request = new RestRequest(string.Concat(ConfigConstants.GetEmployeeURL, employeeId), Method.GET);
+
+            await AddCookiesInRequest(request);
+
+            return await Task.Run(() => client.Get<RestResponse>(request));
+        }
+
+        public async Task<IRestResponse> CreateEmployeeAsync(string employeeName, uint employeeSalary, byte employeeAge)
+        {
+            var client = new RestClient(ConfigConstants.CreateEmployeeURL);
+
+            var request = new RestRequest(ConfigConstants.CreateEmployeeURL, Method.POST);
+
+            await AddCookiesInRequest(request);
+
+            var body = string.Format("{{\"name\":\"{0}\",\"salary\":\"{1}\",\"age\":\"{2}\"}}", employeeName, employeeSalary, employeeAge);
```

Comment:
Consider using serialization instead of interpolation

Reply:
Changes done